client/app/ui-components/button/button.component.js

```diff
@@ -1,75 +1,43 @@
 angular.module("managerApp")
-    .component("cuiButton", {
-        template: `
-            <button class="oui-button oui-button_{{ $ctrl.type }}"
-                    type="submit"
-                    data-ng-bind="$ctrl.text"
-                    data-ng-click="$ctrl.onClick()"
-                    data-ng-disabled="$ctrl.disabled"></button>
-        `,
-        bindings: {
-            text: "<",
-            onClick: "&",
-            disabled: "<",
-            type: "@"
-        }
-    })
     .component("cuiSubmitButton", {
         template: `
-            <button class="oui-button oui-button_primary"
-                    type="submit"
-                    data-ng-bind="$ctrl.text"
-                    data-ng-click="$ctrl.onClick()"
-                data-ng-disabled="$ctrl.disabled"></button>
+            <oui-button variant="primary"
```

Comment:
You should better replace every `cji-submit-button` as this component is replaced by `oui-button`

Reply:
there are many cui-buttons-cancel and cui-submit-button, around 150 entries. Majority of them are used in popup modal. We are currently replacing all cui-modal with oui-modal. This eliminates all these cui buttons and brings count under 30. Then we will replace all remaining cui buttons and delete this component from Cloud manager.

Reply:
Sometimes you stick with `data-` as prefix and sometimes not. Could you please stay uniform? :)